src/metldata/builtin_transformations/common/path/path_utils.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 - 2024 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for 'path' operations"""
+
+from metldata.builtin_transformations.common.path.path import RelationPath
+from metldata.builtin_transformations.common.path.path_elements import (
+    RelationPathElementType,
+)
+from metldata.transform.exceptions import (
+    NotDirectRelationError,
+)
+
+
+def get_referred_class(relation_path: RelationPath) -> str:
```

Comment:
The constraint that we only handle direct relations for now should be reflected in the function name.